src/main/java/io/lbry/browser/LbrynetService.java

```diff
@@ -118,6 +158,188 @@ protected void doStartForeground(Bundle extras) {
+            try {
+                if (reader != null) {
+                    reader.close();
+                }
+                if (conn != null) {
+                    conn.disconnect();
+                }
+            } catch (IOException ex) {
+                // pass
+            }
+        }
+    }
+
+    private void handlePollFileResponse(JSONObject response) {
+        Context context = getApplicationContext();
+        if (response.has("result")) {
+            JSONArray fileItems = response.optJSONArray("result");
+            if (fileItems != null) {
+                try {
+                    List<String> itemUris = new ArrayList<String>();
+                    for (int i = 0; i < fileItems.length(); i++) {
+                        JSONObject item = fileItems.getJSONObject(i);
+                        String downloadPath = item.isNull("download_path") ? null : item.getString("download_path");
+                        if (downloadPath == null || downloadPath.trim().length() == 0) {
+                            continue;
+                        }
+
+                        String claimId = item.getString("claim_id");
+                        String claimName = item.getString("claim_name");
+                        String uri = String.format("lbry://%s#%s", claimName, claimId);
+                        String outpoint = item.getString("outpoint");
+                        boolean completed = item.getBoolean("completed");
+                        double writtenBytes = item.optDouble("written_bytes", -1);
+                        double totalBytes = item.optDouble("total_bytes", -1);
+
+                        if (downloadManager.isDownloadActive(uri) && (writtenBytes == -1 || totalBytes == -1)) {
+                            // possibly deleted, abort the download
+                            downloadManager.abortDownload(uri);
+                            continue;
+                        }
+
+                        if (!itemUris.contains(uri)) {
+                            itemUris.add(uri);
+                        }
+
+                        Intent intent = new Intent();
+                        intent.setAction(DownloadManager.ACTION_DOWNLOAD_EVENT);
+                        intent.putExtra("uri", uri);
+                        intent.putExtra("outpoint", outpoint);
+                        intent.putExtra("file_info", item.toString());
+
+                        File file = new File(downloadPath);
+                        if (downloadManager.isDownloadActive(uri)) {
+                            if (writtenBytes >= totalBytes || completed) {
+                                // completed download
+                                android.util.Log.d("ReactNativeJS", "***");
+                                android.util.Log.d("ReactNativeJS", String.format("Completing download for uri: %s", uri));
+                                android.util.Log.d("ReactNativeJS", String.format("downloadPath=%s", downloadPath));
```

Comment:
Should these log statements be in the release?

Reply:
No. These will be removed.